pkg/controller/migmigration/task.go

```diff
@@ -1313,19 +1313,22 @@ func (t *Task) getPVCs() map[k8sclient.ObjectKey]migapi.PV {
 }
 
 // Get whether the associated plan lists not skipped PVs.
-// First return value is PVs overall, and second is limited to Move or snapshot copy PVs
-func (t *Task) hasPVs() (bool, bool) {
-	var anyPVs bool
+func (t *Task) hasPVs() bool {
 	for _, pv := range t.PlanResources.MigPlan.Spec.PersistentVolumes.List {
-		if pv.Selection.Action == migapi.PvMoveAction ||
-			pv.Selection.Action == migapi.PvCopyAction && pv.Selection.CopyMethod == migapi.PvSnapshotCopyMethod {
-			return true, true
-		}
 		if pv.Selection.Action != migapi.PvSkipAction {
-			anyPVs = true
+			return true
+		}
+	}
+	return false
+}
+
+func (t *Task) isAnyPVsUsingSnapshotMethod() bool {
```

Comment:
This should be renamed to `isAnyPVsUsingMoveOrSnapshotCopyMethod` and the check needs to include `migapi.PvMoveAction ||`

Reply:
Also, either change `are` to `is` in the name, or change `PVs` to `PV`.

Reply:
+1 good catch, I updated the PR with suggested changes